src/averageAges.js

```diff
@@ -18,44 +18,43 @@ function calculateMenAverageAge(people, century) {
   // write code here
   // learn how to use array methods like .filter .map .some .every .find .reduce
   // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
+  // replace if () statement with &&, || or ?:
   // without nesting
+
+  const men = people.filter(person => century
+    ? person.sex === 'm' && Math.ceil(person.died / 100) === century
+    : person.sex === 'm'
+  );
+
+  return men.map(person => person.died - person.born)
+    .reduce((a, b) => a + b) / men.length;
```

Comment:
Map is redudant here. Use only `reduce`.
Also you have similar code in `calculateWomenAverageAge`
So create separate function to `getAvarageAge` and use it here and `calculateWomenAverageAge`